config.py

```diff
@@ -5,7 +5,7 @@
 import json
 
 # Get configuration from environment
-DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:postgres@localhost:5432/postgres")
+DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:password@localhost:5432/postgres")
```

Comment:
@ssr8998 Wondering why we want to change the password of the database?

Reply:
> Seems there are overlapping changes in both branch, restx_suppliers and swagger, not sure which need to be merged first. And the code coverage is dropping below 95%

The swagger is the initial PR and swagger need to be PR first .

Reply:
DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:postgres@localhost:5432/postgres")
DATABASE_URI = os.getenv("DATABASE_URI", "postgres://postgres:password@localhost:5432/postgres")
was a typo and its fixed .